processing-service-core/api/swagger/swagger.yaml

```diff
@@ -952,7 +954,119 @@ paths:
           description: The specified Workflow could not be found.
           schema:
             $ref: "#/definitions/errorMessage"    
+  ##########################
+  ##  Default Config Store
+  ##########################
+  /globalConfig/{key}:
+    x-swagger-router-controller: globalConfig
+    parameters:
+      - name: key
+        in: path
+        type: string
+        maxLength: 255
+        description: The unique identifier for the global configuration.
+        required: true
+    put:
+      tags:
+        - Global Config
+      parameters:
+        - name: globalConfig
+          in: body
+          description: The configuration to use when setting the global config.
+          required: true
+          schema:
+            $ref: "#/definitions/globalConfig"
+      summary: Sets global config for the key.
+      description: Creates a global configuration setting for this key or updates the currently existing global configuration setting if it already exists.
+      operationId: setGlobalConfig
+      responses:
+        204:
+          description:  Indicates that the global configuration was set successfully.
+        400:
+          description: Indicates invalid parameter was passed.
+          schema:
+            $ref: "#/definitions/errorMessage"
+    get:
+      tags:
+        - Global Config
+      summary: Returns the global config for the key.
+      description: Returns the global config for the key.
+      operationId: getGlobalConfig
+      responses:
+        200:
+          description: Returns the global configuration stored for the supplied key.
+          schema:
+            $ref: "#/definitions/globalConfig"
+        400:
+          description: Indicates invalid parameter was passed.
+          schema:
+            $ref: "#/definitions/errorMessage"
+        404:
+          description: Indicates key was not found.
+    delete:
+      tags:
+        - Global Config
+      summary: Deletes global config for the the key.
+      description: Deletes global config for the the key.
+      operationId: deleteGlobalConfig
+      responses:
+        204:
+          description: Deletes the global config for the supplied key.
```

Comment:
Change description to "Indicates the global config for the supplied key was deleted."